storefront/modules/orders/components/order.module.css

```diff
@@ -0,0 +1,133 @@
+table#order {
```

Comment:
@fliptation Maybe you have some feedback about CSS authoring?

Reply:
@fliptation CSS updated and also the order button only visible for users with orders, in addition, to be logged in.